host_v2/src/components/GameInProgressContent.tsx

```diff
@@ -28,30 +29,17 @@ interface QuestionData {
   imageUrl: string | undefined; // the url of the image on the question card (if there is one)
 } // eslint-disable-line
 
```

Comment:
I renamed this file from `PlaceholderContentArea` to `GameInProgressContent`. This replaces `GameInProgressContentSwitch`